api/src/main/java/marquez/api/JobResource.java

```diff
@@ -164,11 +164,13 @@ public Response listJobVersions(
   @Produces(APPLICATION_JSON)
   public Response list(
       @PathParam("namespace") NamespaceName namespaceName,
+      @QueryParam("lastRunState") @DefaultValue("%") String lastRunState,
```

Comment:
We should use enum [`RunState`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/common/models/RunState.java) for input validation and return a `400` on unknown run state types.